apps/server/src/modules/provisioning/config/provisioning-config.ts

```diff
@@ -5,11 +5,13 @@ export const ProvisioningFeatures = Symbol('ProvisioningFeatures');
 export interface IProvisioningFeatures {
 	schulconnexGroupProvisioningEnabled: boolean;
 	schulconnexCourseSyncEnabled: boolean;
+	schulconnexOtherGroupusersEnabled: boolean;
 }
 
 export class ProvisioningConfiguration {
 	static provisioningFeatures: IProvisioningFeatures = {
 		schulconnexGroupProvisioningEnabled: Configuration.get('FEATURE_SANIS_GROUP_PROVISIONING_ENABLED') as boolean,
 		schulconnexCourseSyncEnabled: Configuration.get('FEATURE_SCHULCONNEX_COURSE_SYNC_ENABLED') as boolean,
+		schulconnexOtherGroupusersEnabled: Configuration.get('FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED') as boolean,
```

Comment:
we are not supposed to do it like this anymore. The new way is:
1. Create an interface in the module root, like provisioning-config.ts:
``` ts
export interface ProvisioningConfig {
    FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED: boolean;
}
```
3. Add the interface to the server.config.ts with `extends ProvisioningConfig` and add `FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED: Configuration.get('FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED') as boolean` like you already did.
4. Inject the NestJS `ConfigService<ProvisioningConfig, true>` into your service and use `configService.get('FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED')`
5. Only add features that you need in the nuxt-client to the ConfigResponse (it replaces publicAppConfig.js or however it was called)

Reply:
Maybe we should take this to a refactoring ticket for our existing configuration module :) 
Instead of having both ways

Reply:
What do you say @MarvinOehlerkingCap ?

Reply:
We should have a Ticket for that, since Cedric told us to remove these old interfaces, but I can't find it. We should also finish more of our refactoring tickets! I will bring this topic to the next retro, but for now it is fine i guess.